spec.html

```diff
@@ -16171,20 +16171,19 @@
           <p>The following is an informative definition of an ECMAScript generator function that conforms to these rules:</p>
           <pre><code class="javascript">
             function* EnumerateObjectProperties(obj) {
-              let visited = new Set;
-              for (let key of Reflect.ownKeys(obj)) {
-                if (typeof key === "string") {
-                  let desc = Reflect.getOwnPropertyDescriptor(obj, key);
-                  if (desc && !visited.has(key)) {
-                    visited.add(key);
-                    if (desc.enumerable) yield key;
-                  }
+              const visited = new Set;
+              for (const key of Reflect.ownKeys(obj)) {
```

Comment:
@leobalter, thanks for the review. Is it https://github.com/tc39/ecma262/pull/656/commits/46b569dc9f9ad621c2c97a022eae3d7e8603a3e3 or https://github.com/tc39/ecma262/pull/656/commits/f1883f50a6c7df79902f01c660486b889bdd7b07 that should be dropped?